services/net/filemonitor/FilemonitorAction.cs

```diff
@@ -240,7 +240,7 @@ private async Task ImportArticleAsync(IIngestServiceActionManager manager, Sourc
                 reference = await AddContentReferenceAsync(manager.Ingest, content);
             }
             else if ((reference.Status == (int)WorkflowStatus.InProgress && reference.UpdatedOn?.AddMinutes(5) < DateTime.UtcNow) ||
-                    (reference.Status != (int)WorkflowStatus.InProgress && reference.PublishedOn != content.PublishedOn))
+                    (reference.Status != (int)WorkflowStatus.InProgress && reference.PublishedOn < content.PublishedOn))
```

Comment:
Only post content with a newer published date.  Technically this should be a new updated value, as the published date should not change once content has been published.  However, reviewing the source content we must do this.